src/helix-ui/styles/config.less

```diff
@@ -15,8 +15,3 @@
   0% { transform: rotate(0deg); }
   100% { transform: rotate(360deg); }
 }
-
-:root {
-  --hxSelect__trigger-background: @gray-0;
-  --hxSelect__trigger-background--disabled: @gray-100;
-}
```

Comment:
Removed defaults, because default values are now defined inline with the hook definitions. 

See `src/helix-ui/elements/HXSelectElement.less`.